packages/elements-react/src/components/form/nodes/input.tsx

```diff
@@ -72,7 +72,7 @@ export const NodeInput = ({
     case UiNodeInputAttributesTypeEnum.Submit:
     case UiNodeInputAttributesTypeEnum.Button:
       if (isSocial) {
-        return <Node.OidcButton attributes={attrs} node={node} />
+        return null
```

Comment:
Social buttons are handled by a different form (to prevent form validation issues when submitting the oidc button). So here we have to return null, to not render it. Not ideal, but atm the best I could come up with...